.eslintrc

```diff
@@ -12,6 +12,11 @@
     "node": true,
     "jest/globals": true
   },
+  "globals": {
```

Comment:
should be able to use https://github.com/gajus/eslint-plugin-flowtype/blob/d7ccd0ddefbe6589d60f908c90efe453e7604c50/README.md#eslint-plugin-flowtype-rules-define-flow-type instead

Reply:
Yep, but seems like it's not supported right now.
https://github.com/gajus/eslint-plugin-flowtype/issues/322

Reply:
That's really odd. I tested adding this snippet to my code base:

```js
somethingGlobal();

const foo: TimeoutID = setTimeout(() => {}, 300);
const bar: IntervalID = setInterval(() => {}, 300);
const baz: AnimationFrameID = requestAnimationFrame(() => {});
```

Runnning eslint on that gives:
```
  20:1   error  'somethingGlobal' is not defined          no-undef
  22:7   error  'foo' is assigned a value but never used  no-unused-vars
  22:12  error  'TimeoutID' is not defined                no-undef
  23:7   error  'bar' is assigned a value but never used  no-unused-vars
  23:12  error  'IntervalID' is not defined               no-undef
  24:7   error  'baz' is assigned a value but never used  no-unused-vars
  24:12  error  'AnimationFrameID' is not defined         no-undef
```

Adding `extends: "plugin:flowtype/recommended"` to the eslint config gives:
```
  20:1  error  'somethingGlobal' is not defined          no-undef
  22:7  error  'foo' is assigned a value but never used  no-unused-vars
  23:7  error  'bar' is assigned a value but never used  no-unused-vars
  24:7  error  'baz' is assigned a value but never used  no-unused-vars
```

No guarantee about the type actually existing, but flow deals with that

Reply:
That that this is needed. This was the thing that made me bail on the upgrade myself. Adding globals seems like the right way to go for now until the eslint-flowtype upgradedhttps://github.com/gajus/eslint-plugin-flowtype/issues/322